KnockKnock-iOS/Scenes/Feed/FeedWrite/Views/shopSearchView.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  shopSearchView.swift
+//  KnockKnock-iOS
+//
+//  Created by Daye on 2022/03/20.
+//
+
+import UIKit
+
+import Then
+
+final class shopSearchView: UIView {
```

Comment:
```suggestion
final class ShopSearchView: UIView {
```
요렇게 수정 부탁드립니다!! 타이핑 실수로 보이네요